Library/Homebrew/test/sbom_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require "sbom"
+
+RSpec.describe SBOM, :needs_network do
+  subject(:sbom) { described_class.create(f) }
+
+  let(:f) { formula { url "foo-1.0" } }
```

Comment:
I think adding some dependencies in here would make the test coverage a bit better.